druid/src/widget/label.rs

```diff
@@ -45,13 +56,13 @@ pub enum LabelText<T> {
 #[doc(hidden)]
 pub struct Dynamic<T> {
```

Comment:
Not really related to this PR, but I think it is a bit irritating that `Dynamic` is `doc(hidden)`. It makes the `LabelText` documentation slightly weird, because `Dynamic` looks like a broken link.